webrob/neems/neem.py

```diff
@@ -18,6 +20,8 @@ def __init__(self,
             b_id = bson.objectid.ObjectId(neem_id)
         else:
             b_id = neem_id
+        neemHubSettings = get_settings()
+        mongoDBMetaCollection = getMongoDBMetaCollection(neemHubSettings)
```

Comment:
it seems that at every place you call getMongoDBMetaCollection, get_settings is called before. So I would suggest to do this withing the getMongoDBMetaCollection call